### PR TITLE
Update puppet

### PR DESCRIPTION
**Pull request by annakocheshkova**
- Added extra info on the `AppCenter` tab
image

- Added attachement support on `Crashes` tab
image

- Rearranged `Transmission` tab
image

### CODE REVIEW

File: Assets/AppCenter/Plugins/AppCenterSDK/Core/Shared/AppCenter.cs [hunk 1]
```diff
@@ -144,6 +166,13 @@ public static void SetWrapperSdk()
                                                WrapperSdk.WrapperRuntimeVersion, null, null, null);
         }
 
+        // Gets cached secret.
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/AppCenterSDK/Core/Shared/AppCenter.cs:173]: Use XML-doc

File: Assets/AppCenter/Plugins/AppCenterSDK/Core/Shared/AppCenter.cs [hunk 2]
```diff
@@ -144,6 +166,13 @@ public static void SetWrapperSdk()
...
+        public static AppCenterTask<string> GetSecretForPlatform()
+        {
+            secretTask = new AppCenterTask<string>();
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/AppCenterSDK/Core/Shared/AppCenter.cs]: Should it be created on every call?

File: Assets/Puppet/PuppetAppCenter.cs [hunk 3]
```diff
@@ -14,8 +15,15 @@
...
 {
     public Toggle Enabled;
+    public Text InstallIdLabel;
+    public Text AppSecretLabel;
+    public Text LogUrlLabel;
+    public Text DeviceIdLabel;
+    public Text SdkVersionLabel;
     public Dropdown LogLevel;
     public PuppetConfirmationDialog userConfirmationDialog;
+    public static string TextAttachmentKey = "text_attachment";
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetAppCenter.cs]: `const`

File: Assets/Puppet/PuppetAppCenter.cs [hunk 4]
```diff
@@ -84,6 +92,41 @@ private void Awake()
...
+    public static ErrorAttachmentLog[] GetErrorAttachmentstHandler(ErrorReport errorReport)
+    {
+        return new ErrorAttachmentLog[]
+        {
+            ErrorAttachmentLog.AttachmentWithText(PlayerPrefs.GetString(TextAttachmentKey), "hello.txt"),
+            ErrorAttachmentLog.AttachmentWithBinary(ParseBytes(PlayerPrefs.GetString(BinaryAttachmentKey)), "fake_image.jpeg", "image/jpeg")
+        };
+    }
+
+    private static byte[] ParseBytes(string bytesString)
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetAppCenter.cs:105]: Have no idea how you will test maximum size of binary attachment 🤔

File: build.cake [hunk 5]
```diff
@@ -504,7 +504,7 @@ void BuildApps(string type, string projectPath = ".")
 void VerifyIosAppsBuild(string type, string projectPath)
 {
     VerifyAppsBuild(type, "ios", projectPath,
-    new string[] { "IosMono", "IosIl2CPP" },
+    new string[] { "IosIl2CPP" }, //new string[] { "IosMono", "IosIl2CPP" },
```
Comment (MatkovIvan) [inline build.cake:507]: Isn't related to Puppet

File: Assets/Puppet/PuppetAppCenter.cs [hunk 6]
```diff
@@ -97,17 +140,6 @@ public static bool ShouldProcessErrorReportHandler(ErrorReport errorReport)
...
-//    [MonoPInvokeCallback(typeof(Crashes.GetErrorAttachmentsHandler))]
-//    public static ErrorAttachmentLog[] GetErrorAttachmentstHandler(ErrorReport errorReport)
-//    {
-//        byte[] bytes = new byte[] { 100, 101, 102, 103 };
-//        return new ErrorAttachmentLog[]
-//        {
-//             ErrorAttachmentLog.AttachmentWithText("Hello world!", "hello.txt"),
-//             ErrorAttachmentLog.AttachmentWithBinary(bytes, "fake_image.jpeg", "image/jpeg")
-//        };
-//    }
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetAppCenter.cs]: Remove

**Review by max-mironov — APPROVED** [review]

**Pull request merged**